benefit/data/sheets/FixedLists/FixedLists.json

```diff
@@ -46,6 +46,8 @@
   {"LiveFrom": "01/01/2018", "ID": "documentType", "ListElementCode": "reinstatementRequest", "ListElement": "Reinstatement request"},
   {"LiveFrom": "01/01/2018", "ID": "documentType", "ListElementCode": "urgentHearingRequest", "ListElement": "Urgent hearing request"},
   {"LiveFrom": "01/01/2018", "ID": "documentType", "ListElementCode": "confidentialityRequest", "ListElement": "Confidentiality request"},
+  {"LiveFrom": "01/09/2021", "ID": "documentType", "ListElementCode": "postponementRequest", "ListElement": "Postponement request"},
+  {"LiveFrom": "01/09/2021", "ID": "documentType", "ListElementCode": "postponementRequestDirectionNotice", "ListElement": "Postponement Request direction notice"},
```

Comment:
This is wrong - it will appear in the prod version. It's already in FixedLists-postponement-request-nonprod.json so will appear in the list twice.

Reply:
removed from the prod list